DeeployTest/generateTrainingNetwork.py

```
@@ -0,0 +1,373 @@
+    if len(npz_base) != len(non_grad_indices):
+        raise ValueError(
+            f"inputs.npz has {len(npz_base)} base entries but network.onnx has "
+            f"{len(non_grad_indices)} non-grad-buf inputs. "
+            f"Re-generate inputs.npz with the updated exporter.")
+
+    # Build inputTypes / inputOffsets for ALL graph input positions.
+    inputTypes = {}
+    inputOffsets = {}
+
+    npz_idx = 0
+    for graph_idx, name in enumerate(graph_input_names):
```

Ruff (0.15.9)

[warning] Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)